src/scancode/cli.py

```diff
@@ -313,7 +320,8 @@ def scan(input_path,
          processes=1, timeout=DEFAULT_TIMEOUT, max_memory=DEFAULT_MAX_MEMORY,
          diag=False,
          scans_cache_class=None,
-         strip_root=False):
+         strip_root=False,
+         ignore={}):
```

Comment:
never ever use a mutable as a default arg in a function, unless you know what you are doing. This is a recipe for a catastrophy:  http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments